src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
+@import 'mixins/variables';
+@import 'blocks/stars';
+@import 'blocks/card';
+@import 'blocks/header';
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+h1 {
+  margin: 0;
+}
+
+.container {
+  box-shadow: 0 2px 4px 0 #0000000D;
 }
```

Comment:
It's better to move styles to the separate file to keep the main file clean and clear